### ISSUE TITLE
Why add_audio and add_histogram are disabled within GlobalSummaryWriter?

### ISSUE THREAD

**Issue by iclementine** [lanpa/tensorboardX#592]
Glad to find global writer has been added to tensorboardx. It means alot to me:

With a global writer, easily log without passing the summary writer and global step as parameters of the function within which you want to write logs (as a function to compute values, it should not know about the writer or step), or to alter the return value of the function (to return what you want to log) if you don't want to add extra parameters to that function. And this happens recursively.

But why `add_audio` and `add_histogram` are now disabled for GlobalSummaryWriter? Is there any problem with them now?

**Comment by lanpa** [comment]
Yes, because they require more time to log, comparing to scalar or text. Another reason is that the `step` of the events can not be interleaved otherwise tensorboard will ignore the older ones. It need some memory based IO so that the training program will not be blocked by tensorboardX. btw, does GlobalSummaryWriter work as expected in your program?
